core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -161,8 +165,13 @@ public void serverStarting(Server server) throws Exception {
                 spec = addDocStrings(spec, services);
                 spec = addExamples(spec);
 
+                final Version version = Version.identify(DocService.class.getClassLoader())
+                                               .get(SERVER_ARTIFACT_ID);
```

Comment:
Could we export the `Version` for all artifacts? i.e. you could just serialize the `Map` returned by `identity()`.

Reply:
> Could we export the `Version` for all artifacts? i.e. you could just serialize the `Map` returned by `identity()`.

I have a question. I used `Map` but It is returns `armeria-{module}` versions. I think It's use same version info. Why did you ask me to use it?

Reply:
It is because a user may have a build problem where he or she uses the Armeria JARs of different versions, e.g. `armeria-0.90.0` and `armeria-thrift-0.90.1`. We could show the artifacts whose version numbers are different from that of `armeria` rather than showing all artifact versions, though. Makes sense?

Reply:
Thank you for description.
I changed versions metadata for supporting `armeria-{module}`